deployments/AKS/README.md

```diff
@@ -0,0 +1,12 @@
+# Deploy KubeArmor on AKS
```

Comment:
EKS has some prerequisites. don't we have any prerequisites in the case of AKS?

Reply:
@nam-jaehyun yes. AKS doesn't give any configuration options right now which might affect KubeArmor's functionality.
While trying out KubeArmor on AKS I really haven't faced the need to do anything else.